### DISCUSSION TITLE
SU2_DOT_AD segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by ardao13** [su2code/SU2#1816] [Q&A]
Hi,

I have two computers. One is ubuntu other is centos. They have exact same installation of SU2 7.4.0. The one with Ubuntu can run the following tutorial without any problem.

https://su2code.github.io/tutorials/Inviscid_3D_Constrained_ONERAM6/

However, the other with centos has a problem while running SU2_DOT_AD. As follows:

```
---------- Start gradient evaluation using sensitivity information ----------

Evaluate functional gradient using Algorithmic Differentiation (ZONE 0).
1 Free Form Deformation boxes.
1 Free Form Deformation nested levels.
FFD box tag: WING. FFD box level: 0. Degrees: 10, 8, 1.
FFD Blending using B-Splines. Order: 0, 0, 0. 
Number of parent boxes: 0. Number of child boxes: 4836.
Segmentation fault (core dumped)
```

Thanks

**Comment by NAnand-TUD** [comment]
Hey!

In my experience, this is mostly due to a lack of memory. SU2_DOT_AD is usually the most RAM-consuming process (in my experience). If you are running multicore try single core (more ram per core) else try a PC with more memory.

**Reply by ardao13** [reply]
I will try these suggestions. Thank you

**Reply by MaganS** [reply]
@mardar572 If you solved this problem. Can you explain some points?

**Reply by ardao13** [reply]
I increased the swap memory of my system and the error is disappeared.

**Reply by MaganS** [reply]
Thank you @mardar572 
I was getting this error before. I changed versions of the required packages for compilation as given here. https://github.com/su2code/SU2/discussions/2050#discussion-5278285
It solved my problem.

For me, perhaps it was not a memory issue; otherwise, it should persist after changing versions.
